src/utilities/richTextField.js

```diff
@@ -11,6 +11,7 @@ function createMarkup(storyblokHTML) {
   let markup = sanitizeHtml(Storyblok.richTextResolver.render(storyblokHTML), {
     allowedTags: false, // This allows extra HTML tags to be rendered, e.g., H2, img
     allowedAttributes: false, // This allows custom class names
+    allowVulnerableTags: true, // https://github.com/apostrophecms/sanitize-html/blob/main/CHANGELOG.md#1250
```

Comment:
Added this option to remove the warning logs. See https://github.com/apostrophecms/sanitize-html/blob/main/CHANGELOG.md#1250

As far as I understand we use `RichTextField` for the components we have in Storyblok. The following code might be the cause of the warnings:

https://github.com/SU-SWS/ood_giving_site/blob/6c562399ccd1fb2ffb2174523fa0620d3bb6bc3a/src/components/simple/embedScript.js#L12-L16

![image](https://github.com/SU-SWS/ood_giving_site/assets/35101659/1e45e141-964a-4451-b53d-2daac495003e)

Reply:
🤔 Now why do we need to allow the script tag in wysiwyg content? 

Judging by the comment on the following line script was not intentionally allowed.
`allowedTags: false, // This allows extra HTML tags to be rendered, e.g., H2, img`

However, looking at the docs it doesn't look like there is an opt-out or disallow method and I am confused about what it means by allowing [all tags](https://www.npmjs.com/package/sanitize-html#what-if-i-want-to-allow-all-tags-or-all-attributes) vs [any tag](https://www.npmjs.com/package/sanitize-html#what-if-i-dont-want-to-allow-any-tags). 

Content authors should be using the embed component to add scripts to a page otherwise they just won't work. Can you run a quick test to confirm that adding both a script tag with an src and a script tag with inline code `<script>alert('hey you');</script>` fail to run when entered in a WYSIWYG blok?

Reply:
If they don't work, we won't need to worry about XSS.

Reply:
> However, looking at the docs it doesn't look like there is an opt-out or disallow method and I am confused about what it means by allowing [all tags](https://www.npmjs.com/package/sanitize-html#what-if-i-want-to-allow-all-tags-or-all-attributes) vs [any tag](https://www.npmjs.com/package/sanitize-html#what-if-i-dont-want-to-allow-any-tags).

I poked at this in MA earlier. Seems to be:

- `allowedTags: undefined` results in their default array: https://www.npmjs.com/package/sanitize-html#default-options
- `allowedTags: []` allows no tags (or whatever tags you add to the array)
- `allowedTags: false` allows any/all tags

Not super intuitive. 😐

I'd guess script tags would currently by allowed as-is unless there's more sanitization elsewhere.

Reply:
> Can you run a quick test to confirm that adding both a script tag with an src and a script tag with inline code <script>alert('hey you');</script> fail to run when entered in a WYSIWYG blok?

@sherakama content is coming from Storyblok and I've never used it before (I only signed in before to accept space invitations). How can I add this test script in Storyblok? Thanks.

Thanks @ericbakenhus . In this case, should we use `undefined` for `allowedTags` and `allowedAttributes`?

Reply:
@sherakama @ericbakenhus I removed options from the `sanitizeHtml`. It seems like we don't see the warnings any more.

https://app.netlify.com/sites/adapt-giving/deploys/65569182d3a2171047f0e483